gatsby-theme-landing-page/README.md

```diff
@@ -1,68 +1,70 @@
-        environment: process.env.CONTENTFUL_ENVIRONMENT_ID || "master",
-      },
-    },
-  ],
-};
-```
+    ```js
+    // gatsby-config.js
+    require("dotenv").config();
 
-Create a `.env` file with your Contentful space ID and API keys.
+    module.exports = {
+      plugins: [
+        {
+          resolve: "gatsby-theme-landing-page",
+          options: {
+            spaceId: process.env.CONTENTFUL_SPACE_ID,
+            accessToken: process.env.CONTENTFUL_DELIVERY_ACCESS_TOKEN,
+            environment: process.env.CONTENTFUL_ENVIRONMENT_ID || "master",
+          },
+        },
+      ],
+    };
+    ```
 
-```sh
-# .env
-CONTENTFUL_SPACE_ID="<SPACE_ID>"
-CONTENTFUL_DELIVERY_ACCESS_TOKEN="<ACCESS_TOKEN>"
-```
+3. **Create a `.env` file with your Contentful space ID and API keys.**
 
-Import the content model and demo data to your Contentful space
+    ```sh
+    # .env
+    CONTENTFUL_SPACE_ID="<SPACE_ID>"
+    CONTENTFUL_DELIVERY_ACCESS_TOKEN="<ACCESS_TOKEN>"
+    ```
 
-1. Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
+4. **Import the content model and demo data to your Contentful space**
 
-2. Add your space ID to `contentful/import-demo-config.json` and `contentful/import-model-only-config.json`
+    - Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
 
-```json
-{
-  "spaceId": "SPACE_ID",
-  "contentFile": "./contentful/landing-page-model-and-content.json"
-}
-```
+    - Add your space ID to `contentful/import-demo-config.json` and `contentful/import-model-only-config.json`
 
-3. Import the demo (content model and sample data):
-   `yarn import:contentful:demo`
+      ```json
+      {
+        "spaceId": "SPACE_ID",
+        "contentFile": "./contentful/landing-page-model-and-content.json"
+      }
+      ```
+
+    - Import the demo (content model and sample data):
+       `yarn import:contentful:demo`
 
-(Optional) If you only want to import the content model and not the demo data, use the command: `yarn import:contentful:model`
+    - (Optional) If you only want to import the content model and not the demo data, use the command: `yarn import:contentful:model`
 
-Your Contentful space will now contain the content model used by the starter, along with demo content that demonstrates how to use the various content types and landing page components.
+    Your Contentful space will now contain the content model used by the starter, along with demo content that demonstrates how to use the various content types and landing page components.
 
 ## Content model
 
-![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
+    ![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
```

Comment:
This image isn't appearing properly (seems to be wrapped in a code block) 
![image](https://user-images.githubusercontent.com/1227297/145859831-c2110a6a-28c2-4258-8390-5902bd1d9b6d.png)

Reply:
```suggestion
![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
```

I think this should not be indented. 4 space indent == code block

Reply:
```suggestion
![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
```